Octokit/Models/Request/SearchRepositoriesRequest.cs

```diff
@@ -298,37 +298,66 @@ public override string ToString()
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public class DateRange
     {
+        public const string DateTimePattern = "yyyy-MM-dd'T'HH:mm:ss zzz";
+        public const string DatePattern = "yyyy-MM-dd";
+
         private readonly string query = string.Empty;
 
         /// <summary>
         /// Matches repositories with regards to the <param name="date"/>.
         /// We will use the <param name="op"/> to see what operator will be applied to the date qualifier
         /// </summary>
+        [Obsolete("This constructor doesn't use the time component of the specified DateTime. Please use the overload accepting a DateTimeOffset, which also supports time.")]
         public DateRange(DateTime date, SearchQualifierOperator op)
         {
             switch (op)
             {
                 case SearchQualifierOperator.GreaterThan:
-                    query = string.Format(CultureInfo.InvariantCulture, ">{0:yyyy-MM-dd}", date);
+                    query = string.Format(CultureInfo.InvariantCulture, ">{0:DatePatten}", date);
```

Comment:
Does this compile? 🤔

Reply:
I'm thinking it might be better to just use `ToString(DatePattern, CultureInfo.InvariantCulture)` and string concatenation or interpolation than the current format strings...

Reply:
Wow! I got All Checks Passed "continuous-integration/travis-ci/pr — The Travis CI build passed"

Reply:
Hi @khellang you're thinking something like this? 

`query = $"{from.ToString(DateTimePattern, CultureInfo.InvariantCulture)}..{to.ToString(DateTimePattern, CultureInfo.InvariantCulture)}";`

Reply:
I'm thinking something like `query = FormattableString.Invariant($"{from:DateTimePattern}..{to:DateTimePattern}");` 😄